docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+<summary>personal.yaml</summary>
+
+```yaml
+# Common settings for private developer environments
+base: 'personal'
+
+# All private dev environments go to same cluster
+defaultCluster: terra-integration
+
+# Enable services below, eg:
+# releases:
+#   cromwell:
+#     enabled: true
+```
+
+</details>
+
+Finally, each service to be included has its own `values` override file for our environment:
+For `terra/values/workspacemanager/personal/jcarlton.yaml`, you should verify that [vault](https://www.vaultproject.io/) has secret
+stored at the path described. In my case this had already been taken care of.
+
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+
+ingress:
+  staticIpName: terra-integration-jcarlton-workspace-ip
+```
+
+For `terra/values/crljanitor/personal/jcarlton.yaml`, you need the IP address from the `atlantis apply` call in the previous PR.
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+  configPathPrefix: config/terraform/terra/integration/jcarlton
+serviceIP: 35.223.31.120
+```
+
+For `buffer`, there are additional properties:
+```yaml
+vault:
+  pathPrefix: secret/dsde/terra/kernel/integration/jcarlton
+
+ingress:
+  staticIpName: terra-integration-jcarlton-buffer-ip
+
+buffer:
+  pool:
+    configPath: config/toolsalpha
+
+proxy:
+  whitelist:
+    email: "buffer-client-jcarlton@terra-kernel-k8s.iam.gserviceaccount.com"
+```
+## Creating the namespace
+### `kubectl` Setup
+Interaction with Kubernetes is most often accomplished with `kubectl`. This is a local utility that is packaged
+with Kubernetes. Type `kubectl version` to make sure it's installed.
+```shell script
+$ kubectl version
+```
+<details>
+
+```text
+Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
+Server Version: version.Info{Major:"1", Minor:"17+", GitVersion:"v1.17.14-gke.400", GitCommit:"b00b302d1576bfe28ae50661585cc516fda2227e", GitTreeState:"clean", BuildDate:"2020-11-19T09:22:49Z", GoVersion:"go1.13.15b4", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+</details>
+
+ To connect `kubectl` to the cluster, you must first update `kubeconfig` with the proper credentials. 
+`gcloud container` has a [clusters get-credentials](https://cloud.google.com/sdk/gcloud/reference/container/clusters/get-credentials) option to support exactly this use case. In our case
+we need to specify both the project and zone as well as the cluster name:  
+```shell script
+$ gcloud container clusters get-credentials terra-integration --zone us-central1-a --project terra-kernel-k8s
+```
+```text
+Fetching cluster endpoint and auth data.
+kubeconfig entry generated for terra-integration.
+```
+
+### Making the namespace
+```shell script
+$ kubectl create namespace terra-jcarlton
+```
+```text
+namespace/terra-jcarlton created
+```
```

Comment:
I'm a little surprised this is necessary, but maybe DevOps has been doing it for me this whole time.

Reply:
+1, I didn't have permissions to do this myself when I created mine, I had to ask DevOps to create it for me.

Reply:
I've done this twice now. The trick is to use a mechanical keyboard.